circleguard/result.py

```diff
@@ -1,5 +1,6 @@
 from circleguard.loadable import Replay
-from circleguard.enums import ResultType
+from circleguard.enums import ResultType, Mod
```

Comment:
Why is `Mod` being imported? Doesn't look like it's being used in any changes you made

Reply:
I..don't know how that got here

Reply:
oh I do actually, I originally wrote the conversion in the result class